src/index.tsx

```diff
@@ -1,5 +1,13 @@
 import * as React from "react";
 import { render } from "react-dom";
+import WebFont from "webfontloader";
+
 import { App } from "./App";
 
+WebFont.load({
```

Comment:
we can't use google webfonts (privacy issues), we should host the webfont with the app